core/state/statedb_multicoin_test.go

```diff
@@ -147,16 +146,12 @@ func TestGenerateMultiCoinAccounts(t *testing.T) {
 
 	// Get latest snapshot and make sure it has the correct account and storage
 	snap := snaps.Snapshot(root)
-	snapAccount, err := snap.AccountRLP(addrHash)
+	snapAccount, err := snap.Account(addrHash)
 	if err != nil {
 		t.Fatal(err)
 	}
-	account := new(types.StateAccount)
-	if err := rlp.DecodeBytes(snapAccount, account); err != nil {
-		t.Fatal(err)
-	}
-	if !types.IsMultiCoin(account) {
-		t.Fatalf("Expected SnapAccount to return IsMultiCoin: true, found: %v", types.IsMultiCoin(account))
+	if isMultiCoin := types.IsMultiCoin(snapAccount); !isMultiCoin {
+		t.Fatalf("Expected SnapAccount to return IsMultiCoin: true, found: %v", isMultiCoin)
```

Comment:
(nit) Including the value via %v is redundant as it's always false so it can be hard-coded, and the variable removed.